### ISSUE TITLE
Attribut brand en dessous de origin

### ISSUE THREAD

**Issue by couleurmate** [couleurmate/woo.delivery#4]
`($attribute->get_name() === 'pa_origin', 'pa_brand')` suffirait ?

**Comment by couleurmate** [comment]
nope, j'avais déjà test ce script et j'arrivais avoir l'un ou l'autre, mais pas les deux

**Comment by PasLoin** [comment]
Je pense que c'est réglé via loop/price.php
